sonar-helm-for-iac/main_test.go

```diff
@@ -502,7 +535,38 @@ data:
   org.example.prop=true
 `
 
-	result, _ := evaluateTemplate(converters.NewTemplateSources("test-project/templates/a.yaml", converters.Files{"test-project/templates/a.yaml": template, "values.yaml": values, "Chart.yaml": DefaultChartYaml, "config.properties": config}))
+	result, _ := evaluateTemplate(converters.NewTemplateSources("templates/a.yaml", converters.Files{
+		"templates/a.yaml":  template,
+		"values.yaml":       values,
+		"Chart.yaml":        DefaultChartYaml,
+		"config.properties": config}))
 
 	assert.Equal(t, expected, result)
 }
+
+func Test_tpl_cross_file(t *testing.T) {
+	template := []byte(`
+apiVersion: v1
+kind: ConfigMap
+annotations:
+  checksum: {{ .Files.Get "test-project/templates/b.yaml" | sha256sum }}
+`)
+
+	values := []byte(`
+api: v1
+`)
+
+	bYaml := []byte(`
+apiVersion: v1
+kind: ConfigMap
+`)
+
+	result, err := evaluateTemplate(converters.NewTemplateSources("templates/a.yaml", converters.Files{
+		"templates/a.yaml": template,
+		"values.yaml":      values,
+		"Chart.yaml":       DefaultChartYaml,
+		"templates/b.yaml": bYaml}))
+
+	assert.Equal(t, "\napiVersion: v1\nkind: ConfigMap\nannotations:\n  checksum: e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855\n", result)
```

Comment:
This value is actually incorrect. `.Files` still operate on un-prefixed keys, so actual content of `b.yaml` is not retrieved

Reply:
`.Files` can't read templates in Helm; for future `include` this should be acceptable